pkg/cloudprovider/aws/fleet/instance.go

```diff
@@ -78,3 +81,30 @@ func (p *InstanceProvider) Create(ctx context.Context,
 	}
 	return createFleetOutput.Instances[0].InstanceIds[0], nil
 }
+
+func (p *InstanceProvider) getInstanceIDs(nodes []*v1.Node) []*string {
```

Comment:
nit: order your functions so that they flow more naturally. This is called by TerminateInstances, so I'd put it below. I typically try to order my functions using BFS.

Reply:
Good point.